.github/workflows/unittest.yml

```diff
@@ -42,15 +42,22 @@ jobs:
           PYSPARK_VERSION: "3.0.0"
           ARROW_PRE_0_15_IPC_FORMAT: 0
           PY: "3.7"
-        - config: pyarrow-2.0
-          PYARROW_VERSION: "2.0.0"
+        - config: pyarrow-3.0
```

Comment:
After this is changed, remember to change required check in `branch security` of github settings.
Otherwise the PR checks cannot point to the right Actions checks.
Reference: https://docs.github.com/en/github/administering-a-repository/defining-the-mergeability-of-pull-requests/managing-a-branch-protection-rule

Reply:
@selitvin ^

Reply:
Interesting. Was not aware of this. Thanks!